benefit/data/sheets/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -2073,7 +2073,7 @@
     "ID": "processHearingRecordingRequestJointParty",
     "DisplayContext": "OPTIONAL",
     "LiveFrom": "01/01/2018",
-    "FieldShowCondition": "jointParty=\"Yes\"",
+    "FieldShowCondition": "jointParty = \"Yes\"",
```

Comment:
Do you need this spaced out like this?

Reply:
I thought i'd found a bug so was trying all sorts of stuff to fix. Some of the other show conditions had it like that so i changed to see if made any difference. Turned out it was an issue with test data! Anyway, the point is it doesn't matter if it has spaces or not.